src/crud.js

```diff
@@ -0,0 +1,70 @@
+const taskListDisplay = document.querySelector('#data');
+const inputValue = document.querySelector('.new-task');
+const addButton = document.querySelector('.add-to-list');
+
+let taskList = [];
+
+const display = () => {
+  taskListDisplay.innerHTML = '';
+  taskList = JSON.parse(localStorage.getItem('localItem')) || [];
+  taskList.forEach((element) => {
+    const task = document.createElement('div');
+    task.classList.add('task');
+    task.innerHTML = `
+    <input type="checkbox" class="check" id="check">
+    <input class="edit" type="text" value="${element.description}">
+    <div>
+    <i id="${element.index}" class="fa-solid fa-trash-can"></i>
+    </div>
+    `;
+    taskListDisplay.appendChild(task);
+    const taskList2 = task.children[1];
+    taskList2.addEventListener('change', () => {
+      const listingData = document.querySelector('#data');
+      const arrayList = Array.from(listingData.children);
+      const index = arrayList.indexOf(task);
+      const taskLocal = JSON.parse(localStorage.getItem('localItem'));
+      taskLocal[index].description = taskList2.value;
+      localStorage.setItem('localItem', JSON.stringify(taskLocal));
```

Comment:
- [ ] CRUD functionality is properly implemented! 👌 However, _Index_ property of added item does not match new array length. For instance, if a 5th item is added to the list, the index of that item should equal to 5. Let's improve our code so that it meets this requirement! 😃 

**Mismatch in index property and array length:**
![image](https://user-images.githubusercontent.com/12745474/220672513-002152e1-9e5d-4000-97b5-42cce3ad1f9d.png)